ciao-4.11/contrib/share/doc/xml/correct_periscope_drift.xml

```diff
@@ -54,31 +54,35 @@ verbose=2
          <DESC>
 <VERBATIM>
 Running: correct_periscope_drift
-  version = 0.1
+  version = 0.3
 with parameters:
   infile=pcadf537654279N001_asol1.fits.gz
   evtfile=acisf16659N001_evt2.fits.gz
   outfile=driftcorr_asol1.fits
   verbose=2
-  and ASCDS_INSTALL is /soft/ciao-4.8
+  and ASCDS_INSTALL is /soft/ciao-4.11
 ------------------------------------------------------------
 Fitting a line to the data to get reduced stat errors
 Fitting a polynomial of degree 2 to the data
 Fitting a line to the data to get reduced stat errors
 Fitting a polynomial of degree 2 to the data
 ------------------------------------------------------------
 Fit results
-Events show drift range of 0.24 arcsec in yag axis
-Max absolute correction of 0.14 arcsec for yag axis
-Events show drift range of 0.20 arcsec in zag axis
-Max absolute correction of 0.11 arcsec for zag axis
+        Events show drift range of 0.24 arcsec in yag axis
+        Max absolute correction of 0.14 arcsec for yag axis
+        Events show drift range of 0.20 arcsec in zag axis
+        Max absolute correction of 0.11 arcsec for zag axis
 ------------------------------------------------------------
 Writing out corrected aspect solution file to driftcorr_asol1.fits
-To review fit see correction plots in:
-corr_fit_yag.png
-corr_data_yag.png
-corr_fit_zag.png
-corr_data_zag.png
+        To review fit see correction plots in:
+                corr_fit_yag.png
+                corr_data_yag.png
```

Comment:
I'm fine with not automatically popping up the plots when they are made; especially when the tool output still has this language about where they are. 

 We might want to add some stronger language right in the output (Do not use fit without reviewing plots... etc).